app/controllers/subjects_controller.rb

```diff
@@ -0,0 +1,74 @@
+class SubjectsController < ApplicationController
+  before_action :set_subject, only: [:show, :edit, :update, :destroy]
+
+  # GET /subjects
+  # GET /subjects.json
```

Comment:
I do think all comments should be removed for a more final product, but it is nice to have them as we are getting our MVP setup - especially if we decide to decouple the app.